config/locales/client.en.yml

```diff
@@ -373,6 +373,7 @@ en:
         starts_at: "Starts at"
       upcoming_events_list:
         title: "Upcoming events"
+        custom_title: "%{upcoming_events_title}"
```

Comment:
do we need a i18n string for this?

Reply:
Probably not? I split them to make it easier for me to follow while I worked on the solution.

Reply:
yeah, I'd drop it